### DISCUSSION TITLE
Z2M Multiple Devices Suddenly Gone Offline

### DISCUSSION THREAD

**Discussion by norsemangrey** [Koenkk/zigbee2mqtt#19596] [General]
Hi, I’m using a ZBDongle-E (FW: 6.10.3.0) as coordinator for my Zigbee network and Z2M on Home Assistant. It has been working great now for almost a year, coordinating a network of 50+ devices, but yesterday suddenly 15-20 devices just “fell out”, apparently for no reason. They are reporting as Offline and the log shows multiple “Failed to Ping” warnings.

```
Failed to ping 'Bathroom Floor Thermostat' (attempt 1/1, Read 0x1fff00020000001b/1 genBasic(["zclVersion"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":true,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 46565 - 1 - 88 - 0 - 1 after 10000ms))`
```

I have many different types of devices both powered and with battery and it seem random which devices has gone offline as there are some devices of each type.

I did update the Z2M version the other day to 1.33.2-1 the other day, but everything seemed to be working fine after that until this happened. I have tried restarting Z2M serveral times and I have also rolled back the HA VM with the Z2M addon so that the Z2M version is back to 1.30.3-1, but I still have the same issue.

I don’t have much experience troubleshooting network issues and could use som input on how to proceed and get mye devices back online and what might have caused this.

Edit: It also seems I am not able to re-pair devices. I tried re-pairing a device that has gone offline and it just resulted in it disappearing from the Z2M network altogether. Wouldn't that indicate that there was some connection even though it was showing as unavailable and not responding? Why else would it disappear from the network when I reset the device without doing anything in Z2M?

**Comment by AndreaB321** [comment]
I have the same problem, reloading z2m seems to fix the issue

**Comment by norsemangrey** [comment]
Reload? Do you mean restarting Z2M? I've done that several times without
success.

**Reply by AndreaB321** [reply]
Yes, I meant restarting Z2M. For some reason, since 1.33.2-1 I have had several Z2M add-on crashes when adding new devices, and this sometimes leads to already added devices going offline.

**Comment by jimtng** [comment]
Same here. All my powered zigbee devices went offline: IKEA light bulb, IKEA smart plug, and IKEA repeater. Some (not all) Xiaomi battery devices are offline too but I think it's because they're linked to those IKEA as repeaters.

I've tried restarting zigbee2mqtt, and also power cycled the powered devices to no avail. I've also updated my zigbee dongle P (Sonoff) to the latest coordinator firmware.

**Reply by rosarioscavo** [reply]
how have you solved then?

**Comment by samuele2723** [comment]
i am having new issue since few days, not sure if is related to last Z2m version or what's the other reason. Any luck for your issues?

**Comment by norsemangrey** [comment]
Last time this happened I was able to get the network back up again by powering off every thing in my home (turning off  a circuit breaker) and then, after starting Home Assistant computer, Z2M and the Zigbee coordinator, turned on everything little by little. Not optimal, but it beats re-pairing 70+ devices. After that everything has been working fine for more than half a year...until today, when during the night, everything went offline again. I tried to repeat the procedure I did last time but with no luck. All devices are still offline and I have no idea how to get my network back up again.

**Comment by Beast12** [comment]
Did anyone ever found out why? This is also happening to me all of a sudden. Ran for 4 years without issues. Started happening last week. I've then updated it to version 2. Same thing. Restarting z2m fixes it.

**Reply by norsemangrey** [reply]
I still have no clue why. But I do not think my problem is the same as many others, as most seems (like you) to be able to fix this with a simple restart. Every 4-6 months I lose contact with all my Zigbee devices between 4 and 6 i the morning and the only way I am able to get the network up and running again is to switch of every circuit breaker in the house for an hour or so....turn everything on again and wait for 24 hours or so before the Zigbee devices starts responding again.
